test/acceptance/features/supportExistingOperatorBackedServices.feature

```diff
@@ -445,10 +448,10 @@ Feature: Support a number of existing operator-backed services out of the box
            """
     And File "/bindings/$scenario_id/password" exists in application pod
 
-  @disable-openshift-4.9
```

Comment:
This PR will still run checks on OpenShift 4.9, we should remove this tag after merging [openshift/release#35524](https://github.com/openshift/release/pull/35524).

Reply:
or force merging here with failing check on 4.9

Reply:
This scenario is a flaky one on 4.9, that is why it has been disabled, luckily it passed this time.